src/custom_nodes/east_ocr/nms.hpp

```diff
@@ -147,6 +153,7 @@ inline void nms2(const std::vector<cv::Rect>& srcRects,
         if (neigborsCount >= neighbors && scoresSum >= minScoresSum) {
             resRects.push_back(rect1);
             resScores.push_back(score);
+            resMetadata.push_back(data);
```

Comment:
This and above lines -> why not emplace_back?

Reply:
nms.hpp is external file